changes/1887.feature.md

```diff
@@ -0,0 +1 @@
+add some exceptions when schedule computing session
```

Comment:
This will be logged on release note, so we should consider how users will read this and understand.
Something like "Eager error handling when scheduling compute session" would be readable for readers. We could use chatGPT or ask to other seniors for it !